packages/mobile/src/geth/networkConfig.ts

```diff
@@ -94,6 +95,15 @@ const networkConfigs: { [testnet: string]: NetworkConfig } = {
     pgpnpPubKey: pgpnpPubKeyStaging,
     signMoonpayUrl: signMoonpayUrlStaging,
   },
+  [Testnets.mainnet]: {
+    nodeDir: `.${Testnets.mainnet}`,
+    syncMode: DEFAULT_SYNC_MODE,
+    initiallyForno: FORNO_ENABLED_INITIALLY,
+    blockchainApiUrl: 'https://blockchain-api-dot-celo-mobile-mainnet.appspot.com/',
+    pgpnpUrl: pgpnpUrlProd,
+    pgpnpPubKey: pgpnpPubKeyProd,
+    signMoonpayUrl: signMoonpayUrlStaging, // TODO
```

Comment:
👍  the moonpay auth function is still hosted in `celo-org-mobile` so a prod version will have to be made accessible

Reply:
Yes let's sync today to do this 👍